charts/scaffold/Chart.yaml

```diff
@@ -4,7 +4,7 @@ description: Scaffolding the components of the sigstore architecture
 
 type: application
 
-version: 0.3.21
+version: 0.3.22
```

Comment:
if there is a breaking change we should bump here as well `0.4.0`

Reply:
so fulcio helm chart 0.5.x was cut for fulcio app v0.6.x

looking at fulcio app v0.6.x, there doesnt seem to be any breaking change.
not sure what the reason fulcio was bumped to 0.6.x? https://github.com/sigstore/fulcio/releases/tag/v0.6.0

Reply:
"Change username format" was technically a breaking change - However, it's unlikely anyone was using this format, and we weren't using this in production. But it was a change in the format of a subject

Reply:
Sounds like it's ok to make this upgrade. Could someone PTAL?

Reply:
approved

Reply:
thanks kenny!